### ISSUE TITLE
"OCP\AppFramework\Db\DoesNotExistException" when no (unread) articles exist

### ISSUE THREAD

**Issue by bcutter** [owncloud/news#1028]
## Explain the Problem
OwnCloud log shows ```Error	index	OCP\AppFramework\Db\DoesNotExistException: Did expect one result but found none when executing: query "SELECT `items`.* FROM `*PREFIX*news_items` `items` JOIN `*PREFIX*news_feeds` `feeds` ON `feeds`.`id` = `items`.`feed_id` AND `feeds`.`deleted_at` = 0 AND `feeds`.`user_id` = ? AND `items`.`id` = ? LEFT OUTER JOIN `*PREFIX*news_folders` `folders` ON `folders`.`id` = `feeds`.`folder_id` WHERE `feeds`.`folder_id` = 0 OR `folders`.`deleted_at` = 0 ORDER BY `items`.`id` DESC"; parameters Array ( [0] => Username [1] => 13490 ) ; limit ""; offset ""```

## Steps to Reproduce
I read all articles available, so no unread article exists in no feed anymore. It seems that - on the next cron run (which is used for feed updating) - this results in above printed error.

## System Information
* News app version: 2.0.2
* ownCloud version: 10.0.1.2
* PHP version: 5.6
* Database and version: MySQL 5.5.55
* Browser and version: not relevant
* Distribution and version: not relevant

Can you hide this non-relevant error output in OwnCloud log file? In my opinion it´s not an error, not even a warning - just a description of a current database content status. Nothing worth to log or report. Or do I interpret the error the wrong way? Hope to hear from you experts :-)

**Comment by bcutter** [comment]
Update: It seems this message is NOT relevant to an empty set of articles. I need to revert that. I made some tests and the log entry exists whenever updating feeds with the iOS app "CloudNews" (which currently is the only one supporting OwnCloud News app).

**Comment by BernhardPosselt** [comment]
You need to switch to nextcloud the owncloud version is unmaintained fyi

**Comment by bcutter** [comment]
Can the the NextCloud version of the news app be run under OwnCloud?

**Comment by BernhardPosselt** [comment]
No

**Comment by bcutter** [comment]
So this issue doesn´t exist (anymore) in the NextCloud news app?

**Comment by BernhardPosselt** [comment]
No idea but at least someone will fix it

**Comment by bcutter** [comment]
Good point. Will update this issue so anybody else looking for this will be forwarded to the nextcloud news app issue (if it even exists in that app).
